devtools/gradle/settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = projectName
+rootProject.name = 'quarkus-gradle-plugin'
```

Comment:
projectName is set in gradle.properties, if we change this we should remove that file i guess.

Reply:
I'm a little bit fuzzy on which of these Gradle files is for what exactly... :smile: and I had to change this for the built JAR from the plugin to have a better name than projectName.jar, so I would rather not touch this anymore, if that is acceptable to get at least a first version of this IT in?